.gitattributes

```diff
@@ -1 +1,2 @@
 evals/registry/data/**/*.jsonl filter=lfs diff=lfs merge=lfs -text
+evals/registry/data/atpl_exams/samples.jsonl filter=lfs diff=lfs merge=lfs -text
```

Comment:
I don't think this change is intended?

Reply:
@Ein-Tim No, it was not intended. Apologies but I am very new in making PRs. Should I send another PR without this?

Reply:
I made a new commit reverting the change

Reply:
Looks good now!